packages/shorebird_cli/lib/src/executables/adb.dart

```diff
@@ -51,6 +51,7 @@ class Adb {
       if (deviceId != null) ...['-s', deviceId],
       'shell',
       'monkey',
```

Comment:
I'm so confused why we're using monkey at all.

Reply:
Huh, I guess that's a known way.  I'm used to historically sending an intent: https://www.repeato.app/launching-android-applications-via-adb/